scarlet_extensions/Run/runner.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import scarlet.display
+from scarlet.initialization import build_initialization_coadd
+from ..initialization.detection import makeCatalog
+
+
+class Runner:
+    """ Class that lets scarlet run on a set of `Data` objects
+
+        This class makes running on batches of images easier.
+        It runs on multiple or single observations alike.
+
+        parameters
+        ----------
+        datas: list of Data objects
+            the list of Data objects to run scarlet on
+        model_psf: 'numpy.ndarray' or `scarlet.PSF`
+            the target psf of the model
+        ra_dec: `array`
+            ra and dec positions of detected sources
+    """
+
+    def __init__(self, datas, model_psf, ra_dec = None):
```

Comment:
`datas` -> `data`? That makes more sense and is the same as the private variable that you assign to it in the init.